uk.co.compucorp.civicrm.tasksassignments/tests/phpunit/api/v3/DocumentTest.php

```diff
@@ -432,4 +441,19 @@ private function getCustomFieldColumnName($fieldName) {
 
     return CRM_Utils_Array::value($fieldName, $fieldNames);
   }
+
+  /**
+   * @return array
+   */
+  private function fabricateDocumentType() {
```

Comment:
Should this function be in its own class so that other tests can use it?

Reply:
Good idea, I removed the TaskType fabricator and added a OptionValue fabricator with some convenience methods for document type and task type instead.